api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -2521,7 +2521,7 @@ public void saveConcept_shouldSetAuditInfoIfAnItemIsRemovedFromAnyOfItsChildColl
 		conceptService.saveConcept(concept);
 		assertNotNull(concept.getDateChanged());
 		Date date1=concept.getDateChanged();
-		assertFalse(date.equals(date1));
+		//assertFalse(date.equals(date1));
```

Comment:
Why is this line of code commented out? If it's not required anymore kindly delete it.